spec/support/notifier_support.rb

```diff
@@ -0,0 +1,17 @@
+class TestNotifier
+  mattr_accessor :users_notified_of_new_post, :users_notified_of_new_private_post
+
+  def new_post(post, users)
+    self.users_notified_of_new_post = users
+  end
+
+  def new_private_post(post, users)
+    self.users_notified_of_new_private_post = users
+  end
+
+
+  def self.resetted
+    users_notified_of_new_post = nil
+    users_notified_of_new_private_post = nil
```

Comment:
This needs to return self ( have a `self` at the end) -- it used to -- so that you can use TestNotifier.resetted the same as TestNotifier (cause of the `self`).

I think this is only a short-term fix, but is needed to get tests to pass for now.